docs/user/explanations/design-goals.rst

```diff
@@ -0,0 +1,48 @@
+Design Goals
+============
+
+
+Parity with Ophyd
+-----------------
+
+It should be possible to migrate applications that use ophyd_ to ophyd-async. Meaning it must support:
+
+- Definition of devices
+- Conformity to the bluesky protocols
+- Epics (ChannelAccess) as a backend
+
+Ophyd-async should provide built-in support logic for controlling `the same set of devices as ophyd <https://blueskyproject.io/ophyd/user/reference/builtin-devices.html>`_. 
+
+
+Clean Device Definition
+-----------------------
+
+It should be easy to define devices with signals that talk to multiple backends and to cleanly organize device logic via composition. 
```

Comment:
@coretl I think it's important to expand on this, any ideas?

Reply:
We need to be able to:
- Separate the Device interface from the multiple pieces of logic that might use that Device in a particular way
- Define that Signals of a particular type exist without creating them so backends like Tango or EPICS + PVI can fill them in

Reply:
I have put this in.